package.json

```diff
@@ -58,6 +58,7 @@
   },
   "devDependencies": {
     "@ionic/prettier-config": "^2.0.0",
+    "@jridgewell/sourcemap-codec": "^1.4.14",
```

Comment:
Can you add this to `scripts/license.ts` so that it gets included in our generated license file as a part of the next release?